### PR TITLE
Add a command to see who is on bounty

### PR DESCRIPTION
**Pull request by DiamondPlane**
j45 helped me a bit. (I'm not very good at lua)

### CODE REVIEW

**Review by JostP — CHANGES_REQUESTED** [review]
Something is wrong with code, luacheck has failed. And players who has bounty on his head shouldn't be able to see who is a bounty. Bountied player don't see bounty message on purpose.

**Comment by Panquesito7** [comment]
What message should display if the command runner has the bounty?
I think it should be like "No player with a bounty for now".

File: mods/ctf/ctf_bounties/init.lua [hunk 1]
```diff
@@ -155,3 +155,17 @@ minetest.register_chatcommand("place_bounty", {
 		return true, "Put bounty on " .. target
 	end
 })
+
+minetest.register_chatcommand("bp", {
+         description = "See bountied player and the prize",
+         func = function(name)
+                  if bountied_player then
```
Comment (Panquesito7) [inline mods/ctf/ctf_bounties/init.lua:162]: This will fix the LuaCheck run.
```suggestion
                  if bountied_player then
```

**Comment by LoneWolfHT** [comment]
> What message should display if the command runner has the bounty?
> I think it should be like "No player with a bounty for now".

'There are no bounties at the moment'

**Comment by Toby1710** [comment]
> > What message should display if the command runner has the bounty?
> > I think it should be like "No player with a bounty for now".
> 
> 'There are no bounties at the moment'

👍🏻 Thanks

**Comment by Toby1710** [comment]
I forgot to check the Luacheck. Done

**Comment by Toby1710** [comment]
> Something is wrong with code, luacheck has failed. And players who has bounty on his head shouldn't be able to see who is a bounty. Bountied player don't see bounty message on purpose.

Why shouldn't players see their bounty?

**Comment by Panquesito7** [comment]
> > Something is wrong with the code, luacheck has failed. And players who have a bounty on their head shouldn't be able to see who is a bounty. Bountied players don't see the bounty message on purpose.
> 
> Why shouldn't players see their bounty?

The purpose of the bounty is to be a surprise. So this command shouldn't show the bountied player if there's a bounty or not. Simply tell them that there's no bounty yet.

**Comment by TSafa-23** [comment]
Essentially what I did, I recommend testing to make sure it works before doing anything though

**Comment by DiamondPlane** [comment]
Will make a new PR, I have problem with this.

**Comment by DiamondPlane** [comment]
I decided to close it.

**Pull request closed without merging**